examples/espidf-aws-iot/platformio.ini

```diff
@@ -12,6 +12,7 @@ platform = espressif32
 framework = espidf
 board = esp32dev
 monitor_speed = 115200
+monitor_filters = time, colorize
```

Comment:
Looks like this setting is project specific. Do you mind removing it?

Reply:
I misundersood your point. I feel this is necessary as it correctly interprets the escape codes that the demonstration project outputs. Otherwise the codes make it hard to see what is going on. However it's separate from the E-SDK update so will break this out into another PR yes